DX-1714 add webhook event handler docs (#5586)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-gateway/basic-config-and-security/report-monitor-and-trigger-events/event-webhook-tyk-classic.md

Remember to click **Save** to save your changes.

## Set up a webhook event handler in Tyk Operator

Tyk Operator supports event handler integration for Tyk Classic API Definition. By configuring the `event_handlers` field in ApiDefinition Custom Resource Definition (CRD), you can enable webhooks to be triggered by specific API events. 

The process for configuring webhook event handlers using Tyk Operator is similar to that explained in [Set up a webhook event handler in the Tyk Classic API Definition](#set-up-a-webhook-event-handler-in-the-tyk-classic-api-definition). The example API Definition below enabled event handler by setting `spec.event_handlers`.

```yaml {hl_lines=["14-24"],linenos=true, linenostart=1}
apiVersion: tyk.tyk.io/v1alpha1
kind: ApiDefinition
metadata:
  name: webhook-handler
spec:
  name: webhook-handler
  use_keyless: true
  protocol: http
  active: true
  proxy:
    target_url: http://httpbin.org
    listen_path: /webhook-handler
    strip_listen_path: true
  event_handlers:
    events:
      AuthFailure:
      - handler_name: "eh_web_hook_handler"

>         handler_meta:
>           method: "POST"
>           target_path: "http://posttestserver.com/post.php?dir=tyk-event-test"

Reviewer: let's also add webhook name here
```suggestion
        handler_meta:
          method: "POST"
          name: "webhook name"
          target_path: "http://posttestserver.com/post.php?dir=tyk-event-test"
```

Note that you can register multiple webhooks to be triggered in response to a single event and you can register the same webhook with multiple API events.

Remember to click **Save** to save your changes.

## Set up a webhook event handler in Tyk Operator

> Tyk Operator supports event handler integration for Tyk Classic API Definition. By configuring the `event_handlers` field in ApiDefinition Custom Resource Definition (CRD), you can enable webhooks to be triggered by specific API events. 

Reviewer 2: ```suggestion
Tyk Operator supports event handler integration for Tyk Classic API Definition. Configuring the `event_handlers` field in ApiDefinition Custom Resource Definition (CRD) enables webhooks to be triggered by specific API events. 
```

> The process for configuring webhook event handlers using Tyk Operator is similar to that explained in [Set up a webhook event handler in the Tyk Classic API Definition](#set-up-a-webhook-event-handler-in-the-tyk-classic-api-definition). The example API Definition below enabled event handler by setting `spec.event_handlers`.

Reviewer 2: ```suggestion
The process for configuring webhook event handlers using Tyk Operator is similar to that explained in [Set up a webhook event handler in the Tyk Classic API Definition](#set-up-a-webhook-event-handler-in-the-tyk-classic-api-definition). The example API Definition below enables the event handler by setting `spec.event_handlers`.
```
